lib/components/edit-poll/Statements.tsx

```diff
@@ -33,7 +33,7 @@ export default function PollStatements({
       />
       <div className="ml-auto mt-6">
         <Button
-          title="Publish poll"
+          title={poll.published ? "Update poll" : "Publish poll"}
           onClick={async () => {
             await publishPoll(poll.uid, statements);
             router.push(`/${poll.uid}/${poll.urlSlug}`);
```

Comment:
Aside: We should migrate this to `pollUrl` also.

Reply:
I made this update but am unable to test it due to the Anthropic key issue. Please test it if you can!